convert/jenkinsjson/convert.go

```diff
@@ -629,13 +629,35 @@ func collectStepsWithID(currentNode jenkinsjson.Node, stepWithIDList *[]StepWith
 
 	case "readMavenPom":
 		*stepWithIDList = append(*stepWithIDList, StepWithID{Step: jenkinsjson.ConvertReadMavenPom(currentNode), ID: id})
-	
+
 	case "jiraSendBuildInfo":
 		*stepWithIDList = append(*stepWithIDList, StepWithID{Step: jenkinsjson.ConvertJiraBuildInfo(currentNode, variables), ID: id})
 
 	case "jiraSendDeploymentInfo":
 		*stepWithIDList = append(*stepWithIDList, StepWithID{Step: jenkinsjson.ConvertJiraDeploymentInfo(currentNode, variables), ID: id})
 
+	case "s3Upload":
+		entries := jenkinsjson.ExtractEntries(currentNode)
+		if entries == nil {
+			fmt.Println("No entries exists")
+			break
+		}
+		// Iterate over each entry and handle based on the 'symbol' type
+		for _, entry := range entries {
+			gzipFlag, ok := entry["gzipFiles"].(bool)
+			if !ok {
+				fmt.Println("Operation gzipFlag not found or not a boolean")
+				continue
```

Comment:
Doing that, you are skipping one conversion entry, assume gzipFlag is false on that case and proceed.

Reply:
taken care the comment